### ISSUE TITLE
Always use Proguard (in debug too!)

### ISSUE THREAD

**Issue by dral3x** [futurice/android-best-practices#182]
From my experience, is not good having too different configurations between debug and release, and this is especially true when Proguard is enabled only in one.

We found safer have `minifyEnabled true` also for the debug build, and turn off just the obfuscation (with a specific `proguard-rules-debug.pro` file).

You don't want to have your app running fine on your device while crashing on the user phone because you miss something in ProGuard configuration.

**Comment by rpattabi** [comment]
Makes sense. Only contention is the increased build time depending on the size of the project and dependencies.

**Comment by dral3x** [comment]
@rpattabi that's true, I agree.
But since we're not in the javascript world, I prefer the "better safe than sorry" route
